openshift/console
# CONSOLE-3237: Console deletes operands regardless of interdependency

Console deletes operands regardless of interdependency.
This is built from dtaylor's [PR10279](https://github.com/openshift/console/pull/10279)

Added the ability for an operator to "opt out" of deleting all operands using an annotation flag:  console.openshift.io/disable-operand-delete: "true"

Steps to test:
1) Using Console UI, create project "test-3237" and then go to Operator Hub and install Business Automation in namespace "test-3237"
2) Go into operator details for business automation and create instance for kieapps
3) From the operators installed page, use the kebab and select "uninstall operator"
4) If the opt out flag is not set, then the new screens showing instructions to delete all operands will appear
4a) If the opt out flag is set, then the uninstall should look and work as it previously did
5) To change the opt out flag, use the following patch cmds:
oc annotate csv businessautomation-operator.7.13.1-1 console.openshift.io/disable-operand-delete=true
oc annotate csv businessautomation-operator.7.13.1-1 console.openshift.io/disable-operand-delete=false --overwrite

@jhadvig @dtaylor113
